### DISCUSSION TITLE
Sort stories by date

### DISCUSSION THREAD

**Discussion by justified1gmail** [Videogamers0/StoryManager#11] [Ideas]
This has become my favorite way to read Literotica. 

Many times I find my way to an author's page of stories and I want to read them as they were written in chronological order, but the website doesn't have an option to sort them.  If this app could allow them to be sorted by date, then it would certainly make it easier, especially for the more prolific authors.

If the user has the stories grouped by author then the story date option would be a secondary sort, within the author group.

**Comment by Videogamers0** [comment]
7120463081241bfe2c92aafd9626f1e29c61fcab

Sure, sounds good. There's now a couple sorting options in the Settings window for [Version 1.0.5](https://github.com/Videogamers0/StoryManager/releases). If Grouping is enabled, groups are still sorted by author name, but the new sorting options now affect the order of stories within a group. For multi-chapter stories, the date is determined by either the authored date of the first chapter or the last chapter that you've downloaded for each particular series.
